packages/yoastseo/src/parse/build/private/getTextElementPositions.js

```diff
@@ -44,25 +44,38 @@ function getDescendantPositions( descendantNodes ) {
 function adjustElementEnd( descendantNodes, descendantTagPositions, textElementStart, textElementEnd ) {
```

Comment:
```suggestion
function adjustElementEnd( descendantTagPositions, textElementStart, textElementEnd ) {
```

`descendantNodes` looks like it's not being used in this function. The JSDoc parameter should also be adjusted